platform-ui-skin/pom.xml

```diff
@@ -37,6 +37,7 @@
             <plugin>
                 <groupId>org.lesscss</groupId>
                 <artifactId>lesscss-maven-plugin</artifactId>
+                <version>1.7.0.1.1</version>

```

Comment:
Maven plugins version must not be defined directly in modules pom, they are already defined by the parent pom.
Do you really need to upgrade the version of this plugin for your fix ?